packages/docs/src/components/content/MaturityChecklist/MaturityChecklist.tsx

```diff
@@ -1,93 +1,222 @@
+          status: props.items['a11yStandards'],
+        },
+        {
+          title: 'Screen readers',
+          description:
+            'VoiceOver, NVDA, and JAWS screen readers provide concise communication and interaction.',
+          status: props.items['screenReaders'],
+        },
+        {
+          title: 'Keyboard navigation',
+          description: 'Component is fully navigable with a keyboard.',
+          status: props.items['keyboardNavigable'],
+        },
+      ],
+    },
+    {
+      title: 'Code',
+      items: [
+        {
+          title: 'Storybook',
+          description: 'Component has stories to cover all defined props.',
+          status: props.items['storybook'],
+        },
+        {
+          title: 'Responsive',
+          description: 'Component designed to work in all responsive breakpoints.',
+          status: props.items['responsive'],
+        },
+        {
+          title: 'Spanish translations',
+          description: 'Includes Spanish translations for default text content.',
+          status: props.items['spanish'],
+        },
+      ],
+    },
+    // {
+    //   title: 'Design',
+    //   items: [
+    //     {
+    //       title: 'Sketch UI-kit',
+    //       description: 'Includes all Sketch symbols for defined options.',
+    //       status: props.items['completeUiKit'],
+    //     },
+    //     {
+    //       title: 'Responsive',
+    //       description: 'All Sketch symbols designed for small and large breakpoints.',
+    //       status: props.items['responsiveUiKit'],
+    //     },
+    //   ],
+    // },
+    {
+      title: 'Tokens',
+      items: [
+        {
+          title: 'Code',
+          description: 'Tokens implemented in code.',
+          status: props.items['tokensInCode'],
+        },
+        {
+          title: 'Design',
+          description: 'Tokens implemented in the Sketch.',
+          status: props.items['tokensInSketch'],
+        },
+      ],
+    },
+  ];
+
+  // Calculates percentage of completed criteria
+  const progress = () => {
+    let totalAvailableCriteria = 0;
+    let completedCriteria = 0;
+
+    criteria.forEach((criterion) => {
+      criterion.items.forEach((item) => {
+        if (item.status !== null) {
+          totalAvailableCriteria++;
+          if (item.status === true) {
+            completedCriteria++;
+          }
+        }
+      });
+    });
+
+    return Math.round((completedCriteria / totalAvailableCriteria) * 100);
+  };
+
+  // Creates list of incomplete criteria
+  const incompleteCriteria = criteria.map((criterion) => {
+    return criterion.items.map((item) => {
+      if (item.status === false) {
+        return (
+          <MaturityChecklistItem key={item.title} title={item.title} status={item.status}>
+            {item.description}
+          </MaturityChecklistItem>
+        );
+      }
+    });
+  });
+
+  return (
+    <section className="c-maturity-checklist">
+      {props.children}
```

Comment:
i'd like to have hard-coded the "Component maturity" heading here, but Gatsby loses awareness of this heading when it's embedded in a component outside of the content MD file, so it doesn't populate in the table of contents. perhaps i should add a TODO for when we do a future conversion to another SSG so we know why this is weird.